packages/expo-image/ios/EXImageViewManager.m

```diff
@@ -2,11 +2,21 @@
 
 #import <expo-image/EXImageViewManager.h>
 #import <expo-image/EXImageView.h>
+#import <SDWebImageSVGKitPlugin/SDImageSVGKCoder.h>
+#import <SDWebImage/SDImageCodersManager.h>
 
 @implementation EXImageViewManager
 
 RCT_EXPORT_MODULE(ExpoImage)
 
++ (void)initialize
+{
+  SDImageSVGKCoder *svgCoder = [SDImageSVGKCoder sharedCoder];
+  if (![[SDImageCodersManager sharedManager].coders containsObject:svgCoder]) {
+    [[SDImageCodersManager sharedManager] addCoder:svgCoder];
+  }
+}
+
```

Comment:
Do you think it makes sense to make this optional? E.g. only import `SDWebImageSVGKitPlugin` and initialize when the dependency is installed?
```suggestion
#if __has_include(<SDWebImageSVGKitPlugin/SDImageSVGKCoder.h>)
#import <SDWebImageSVGKitPlugin/SDImageSVGKCoder.h>
#import <SDWebImage/SDImageCodersManager.h>
#endif

@implementation EXImageViewManager

RCT_EXPORT_MODULE(ExpoImage)

+ (void)initialize
{
#if __has_include(<SDWebImageSVGKitPlugin/SDImageSVGKCoder.h>)
  SDImageSVGKCoder *svgCoder = [SDImageSVGKCoder sharedCoder];
  if (![[SDImageCodersManager sharedManager].coders containsObject:svgCoder]) {
    [[SDImageCodersManager sharedManager] addCoder:svgCoder];
  }
#endif
}

```

Reply:
I will go with OS version `if` as per https://github.com/expo/expo/pull/7385#discussion_r396287565.

Reply:
As per Slack discussion I have made `SVGKitPlugin` optional in (b7dd5f0533cf111e681a6ec4dc438de864f5e6f1).